packages/mgt-chat/src/components/ChatHeader/Teams.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * -------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.
+ * See License in the project root for license information.
+ * -------------------------------------------------------------------------------------------
+ */
+
+import type { LinkProps } from '@fluentui/react-components';
+import { Link, makeStyles } from '@fluentui/react-components';
+import { MoreHorizontal24Filled, MoreHorizontal24Regular, bundleIcon } from '@fluentui/react-icons';
+import React, { memo } from 'react';
+
+const TeamsEllipsisIcon = bundleIcon(MoreHorizontal24Filled, MoreHorizontal24Regular);
+
+interface TeamsProps {
+  as?: 'a';
+  link: string;
+}
+
+const linkStyles = makeStyles({
+  center: {
+    justifyContent: 'center',
+    display: 'flex'
+  }
+});
+
+export const Teams = (props: LinkProps & TeamsProps) => {
+  const styles = linkStyles();
+
+  return (
+    <Link className={styles.center} {...props} href={props.link} target="_blank">
```

Comment:
I though this was supposed to be an flyout menu triggered by clicking on the ellipsis with a link to Teams.

But looking at the designs I can't see the activated state for this button anywhere @yejuntak can you comment here please?

Reply:
<img width="501" alt="Screenshot 2023-11-02 at 1 00 04 AM" src="https://github.com/microsoftgraph/microsoft-graph-toolkit/assets/112977858/cc9f755b-8a4c-4f49-81fc-8baeb1fd0971">

Are we talking about this? ellipsis button would lead to link to Teams

Reply:
I totally missed this one @yejuntak

Reply:
@yejuntak I can't find that design at all, can you please link me to it?

Reply:
https://www.figma.com/file/gP4q8VQ2so2ftzKz4GCuVh/Microsoft-Graph-Toolkit-(WIP)?type=design&node-id=11213-30069&mode=dev